### ISSUE TITLE
Helis start smoking again after one shot after using a repair kit

### ISSUE THREAD

**Issue by NoxTrap** [echoroleplay/feedback#217]
**Description:** When helicopters (specifically PD, not sure about others) get shot a lot, they start smoking and eventually the engine light turns on and the engine breaks. You can use a repair kit at any point after the heli starts smoking to get it to stop smoking, but it doesn't stop the damaged sounds (the high pitched whining sound) and if it gets shot again (even 1 time), it will immediately start smoking again. 

**Reproduction Steps:** Clearly describe the steps to reproduce the bug. 
1. Shoot a PD heli until it starts smoking
2. Repair the heli with an advanced repair kit
3. Shoot the heli one time and it will start smoking

**Expected Behavior:** The heli is repaired fully, or at least to some point above one shot causing it to start smoking again

**Actual Behavior:** Any heli repaired with an advanced repair kit will begin smoking again after one shot. Right now, PD is not supposed to use the F1 repair on the helipad to fix helis during scenes which is the only thing that can stop this from happening. It's understandable why they can't, but the only alternative is to use a repair kit which doesn't work as well as it does for cars. Even if they don't fully repair helis, it would be great if the heli could at least take half of its normal shots before smoking again and forcing the pilot to land.

**Screenshot(s)/Video(s):** N/A

**Comment by xxmustardxx** [comment]
Repair kits are not meant to be full body repairs lol, they do the same for cars, they are more of a temporary fix to get you to the mechanic, that is pretty op if you ask me being able to just land anywhere after being shot and asking repair kit which takes 90 seconds I think faster if you drink coffee to be full durability again. Cars that have the “broken” sound keep it after repair kits also, need to have a body repair taken to a mechanic, this would be way to powerful .

**Comment by NoxTrap** [comment]
> Repair kits are not meant to be full body repairs lol, they do the same for cars, they are more of a temporary fix to get you to the mechanic, that is pretty op if you ask me being able to just land anywhere after being shot and asking repair kit which takes 90 seconds I think faster if you drink coffee to be full durability again. Cars that have the “broken” sound keep it after repair kits also, need to have a body repair taken to a mechanic, this would be way to powerful .

I agree that a full repair would be too much, but having a heli only take one shot before being immediately unusable is not enough. You can only see people from so far so helis have to get in range of being shot multiple times, especially considering the range on most rifles. Having a helicopter become unusable for the rest of a shootout that is already difficult with a helicopter isn't great. I'm not necessarily asking for the helicopter to be fully repaired with repaired internals, just a couple extra shots so it can continue to do its job.

**Comment by xxmustardxx** [comment]
like i said before repair kits are just to hold you over to get to the mechanic safety, you should not be able to just land anywhere after getting your engine shot so much to the point its red and smoking be able to land, use a car repair kit and in a 1 min 30 seconds, with a car repair kit be able to continue the shootout/chase. that seems a little crazy

**Comment by xxmustardxx** [comment]
not to mention air 1 can fly out of shooting range so quick, that thing moves and turns so quick, if repair kits, are going to fix the helis more needs to be at least 3 times longer in my opinion.
